src/actions/base.ts

```diff
@@ -174,9 +174,10 @@ export abstract class BaseAction {
 export abstract class BaseCommand extends BaseAction {
   /**
    * If isCompleteAction is true, then triggering this command is a complete action -
-   * that means that we'll go and try to run it.
+   * that means that after the command is ran we can clear recordedState.count because
+   * it will have been handled already.
    */
-  isCompleteAction = true;
+  isCompleteAction = false;
```

Comment:
Why change the default value of `isCompleteAction`?

Reply:
Because of the explanation above. When it `isCompleteAction` the action is supposed to handle the `recordedState.count` (either by the default way, which calls the `exec` a total of `recordedState.count` times or by handling it in a custom way on `execCount`).

This was supposed to remember people that if the action isComplete than you have to set that to `true` and then handle `execCount` or let it be handled by default.

If you think this was better the other way, I can change it...